src/index.html

```diff
@@ -5,9 +5,56 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body id="home" class="body">
+
+    <main class="card" data-qa="card">
+
+      <a href="#image" class="card__img"></a>
```

Comment:
1. no needs to add an empty line between parent tag and child one
2. you should use `article` tag for the card
3. you should use `img` tag for the image
4. body shouldn't have `id` and `class` (you can apply styles directly to `body` tag since we have only one `body`)